capplet/gsm-properties-dialog.c

```diff
@@ -217,6 +216,14 @@ _app_removed (GsmPropertiesDialog *dialog,
         gtk_list_store_remove (dialog->list_store, &iter);
 }
 
+static void
+_app_removed (GsmPropertiesDialog *dialog,
+              GspApp              *app,
+              GspAppManager       *manager)
```

Comment:
```
gsm-properties-dialog.c:222:36: warning: unused parameter 'manager' [-Wunused-parameter]

              GspAppManager       *manager)
```
I know `_app_removed` was there before, but now you write  the code snipset new.

Reply:
Ok, I updated the pr because i think it's easier to use a function than a column for setting the visibility of the treeview rows.

Reply:
The warning is still there.
https://travis-ci.org/github/mate-desktop/mate-session-manager/jobs/767534724#L2537
```
gsm-properties-dialog.c:218:36: warning: unused parameter 'manager' [-Wunused-parameter]

              GspAppManager       *manager)
```

Reply:
done

Reply:
Thx